iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -433,6 +454,14 @@ TEST_F(PoshRuntime_test, CreateNodeReturnValue)
     // EXPECT_EQ(nodeDeviceIdentifier, nodeData->m_nodeDeviceIdentifier);
 }
 
+TEST_F(PoshRuntime_test, CreateNodeError)
```

Comment:
This test is not working if debug argument is not used to build and run the test.